edx_event_bus_kafka/config.py

```diff
@@ -5,14 +5,24 @@
 import warnings
 from typing import Optional
 
-from confluent_kafka.schema_registry import SchemaRegistryClient
 from django.conf import settings
 
+try:
+    import confluent_kafka
+    from confluent_kafka.schema_registry import SchemaRegistryClient
+except ImportError:
+    confluent_kafka = None
 
-def create_schema_registry_client() -> Optional[SchemaRegistryClient]:
+
+# return type (Optional[SchemaRegistryClient]) removed for better error messaging when confluent-kafka is not available
```

Comment:
May want to convert to a "Returns" section of the docstring.

Reply:
(Same for other instances of this.)

Reply:
I'm going to put this comment back in so future developers don't blindly add the return type to the method signature and mess up the error messaging